<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>751</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Cutted Cone:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Bottom Base Radius:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="coneTopRadius"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Top Base Radius:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="coneBaseRadius"/>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="coneHeights"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Center Dot:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QDoubleSpinBox" name="cendotX">
               <property name="minimum">
                <double>-200.000000000000000</double>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="cendotY">
               <property name="minimum">
                <double>-200.000000000000000</double>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="cendotZ">
               <property name="minimum">
                <double>-200.000000000000000</double>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Draw</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Prism:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Heights:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="prismHeights"/>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Top Base Coordinates:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Dot 1:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QDoubleSpinBox" name="dot1X">
               <property name="minimum">
                <double>-200.000000000000000</double>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="dot1Y">
               <property name="minimum">
                <double>-200.000000000000000</double>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="dot1Z">
               <property name="minimum">
                <double>-200.000000000000000</double>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Dot 2:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QDoubleSpinBox" name="dot2X">
             <property name="minimum">
              <double>-200.000000000000000</double>
             </property>
             <property name="maximum">
              <double>200.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="dot2Y">
             <property name="minimum">
              <double>-200.000000000000000</double>
             </property>
             <property name="maximum">
              <double>200.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="dot2Z">
             <property name="minimum">
              <double>-200.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Dot 3:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QDoubleSpinBox" name="dot3X">
             <property name="minimum">
              <double>-200.000000000000000</double>
             </property>
             <property name="maximum">
              <double>200.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="dot3Y">
             <property name="minimum">
              <double>-200.000000000000000</double>
             </property>
             <property name="maximum">
              <double>200.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="dot3Z">
             <property name="minimum">
              <double>-200.000000000000000</double>
             </property>
             <property name="maximum">
              <double>200.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Dot 4:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QDoubleSpinBox" name="dot4X">
             <property name="minimum">
              <double>-200.000000000000000</double>
             </property>
             <property name="maximum">
              <double>200.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="dot4Y">
             <property name="minimum">
              <double>-200.000000000000000</double>
             </property>
             <property name="maximum">
              <double>200.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="dot4Z">
             <property name="minimum">
              <double>-200.000000000000000</double>
             </property>
             <property name="maximum">
              <double>200.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Draw</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="TheGLWidget" name="openGLWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>dot1X</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TheGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>theglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>coneTopRadius</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>coneBaseRadius</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>cendotZ</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>cendotX</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>cendotY</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
